app/src/main/java/ch/epfl/sdp/blindly/user/User.kt

```diff
@@ -81,7 +83,7 @@ class User private constructor(
          * @param location the location of the User
          */
         fun setLocation(location: List<Double>) = apply {
-            if(location.size == SIZE_2)
+            if (location.size == SIZE_2)
```

Comment:
I wrote that in a different PR but I still believe SIZE_2 defeats the purpose of making it a constant.

Reply:
True ! I changed it.